src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -120,14 +119,14 @@ public virtual void WriteMessage(LogLevel logLevel, string logName, int eventId,
                 logLevelColors = GetLogLevelConsoleColors(logLevel);
                 logLevelString = GetLogLevelString(logLevel);
                 // category and event id
-                logIdentifier = _loglevelPadding + logName + "[" + eventId + "]";
+                sb.AppendLine(_loglevelPadding + logName + "[" + eventId + "]");
```

Comment:
If it was you would only save a couple milliseconds every few hundred thousand calls to log because most of the time is spent on the lock so any major perf gains will be gained from reducing time spent inside the lock.